pg_store_plans.c

```diff
@@ -1494,6 +1497,9 @@ pg_store_plans_internal(FunctionCallInfo fcinfo,
 			else
 				pstr = SHMEM_PLAN_PTR(entry);
 
+			/* keep compiler quiet */
+			mstr = pstr;
```

Comment:
nitpicking, but this can be set in default branch of the next switch() statement instead, which should avoid an extraneous assignment most of the cases.

Reply:
Right. Fixit that.